js/client-mainmenu.js

```diff
@@ -996,6 +1001,8 @@
 				return '';
 			case 'raw':
 				return '<div class="chat">' + BattleLog.sanitizeHTML(target) + '</div>';
+			case 'nonotify':
+				return {message: '<div class="chat">' + BattleLog.sanitizeHTML(target) + '</div>', noNotify: true};
```

Comment:
I'm normally very opposed to such a weird return value, but I guess it's only in the old client.